packages/prisma/schema.prisma

```diff
@@ -237,13 +237,15 @@ model Credential {
   paymentStatus     String?
   billingCycleStart Int?
 
-  destinationCalendars   DestinationCalendar[]
-  selectedCalendars      SelectedCalendar[]
-  invalid                Boolean?              @default(false)
-  CalendarCache          CalendarCache[]
-  references             BookingReference[]
-  delegationCredentialId String?
-  delegationCredential   DelegationCredential? @relation(fields: [delegationCredentialId], references: [id], onDelete: Cascade)
+  destinationCalendars           DestinationCalendar[]
+  selectedCalendars              SelectedCalendar[]
+  invalid                        Boolean?               @default(false)
+  CalendarCache                  CalendarCache[]
+  references                     BookingReference[]
+  delegationCredentialId         String?
+  delegationCredential           DelegationCredential?  @relation(fields: [delegationCredentialId], references: [id], onDelete: Cascade)
+  calendarSubscriptions          CalendarSubscription[]
```

Comment:
DevinAI, A Calendar Subscription should be only related to a single SelectedCalendar. Each selected calendar should only be able to have a single active subscription.